src/MoBi.Presentation/Presenter/EditParameterPresenter.cs

```diff
@@ -329,5 +337,11 @@ public override bool CanClose
             return !_view.HasError && _editValueFormulaPresenter.CanClose;
          }
       }
+
+      public void Handle(ParameterChangedEvent eventToHandle)
+      {
+         if(eventToHandle.Parameters.Contains(_parameter))
+            _view.Show(_parameterDTO);
```

Comment:
looks weird to me. If any parameter in the list is changed, you refresh a single one?

Reply:
I think this says that the list of parameters in this event contains the parameter being edited then refresh no?